File: selenium_ex/instagram_test/instagram_POM/pages/login_page.py
from selenium.webdriver.common.by import By



class LoginPage:


    def __init__(self,driver):
        self.driver=driver
        self.username_textbox=(By.NAME, "username")
        self.password_textbox=(By.NAME, "password")
        self.login_button=(By.XPATH, '//*[@id="loginForm"]/div/div[3]/button')

File: selenium_ex/TestProject/TestProject1/Business/Pages/LoginPage.cs
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TestProject1.Pages
{
    public class LoginPage
    {
        public readonly IWebDriver _driver;

        public LoginPage(IWebDriver driver)
        {
            _driver = driver;
        }

        public IWebElement UserNameInput => _driver.FindElement(By.Name("username"));
        public IWebElement PasswordInput => _driver.FindElement(By.Name("password"));
        public IWebElement LoginButton => _driver.FindElement(By.XPath("//*[@id=\"loginForm\"]/div/div[3]/button"));
      
    }
}
